manual/thread_safety/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Thread safety
+layout: single
+sidebar:
+  nav: "manual"
+toc: true
+toc_sticky: true
+---
+
+ROOT classes support one of the following thread safety levels:
+
+- Thread unsafe: make sure that any usage of thread unsafe objects is serialized.
+You should not use thread unsafe objects concurrently by multiple threads even if every thread uses its own copy of an object and even if the threads use objects of _different_ (thread unsafe) types.
```

Comment:
What do you mean/refer-to with:

> even if the threads use objects of _different_ (thread unsafe) types.

Reply:
It picks up Enrico's comment: if thread A uses an object of type TX and thread B uses one of type TY, they might still access the same global state (e.g. one inherited from the same base class).